pulsar-client-cpp/lib/ProducerConfigurationImpl.h

```diff
@@ -33,6 +33,7 @@ struct ProducerConfigurationImpl {
     unsigned int batchingMaxMessages;
     unsigned long batchingMaxAllowedSizeInBytes;
     unsigned long batchingMaxPublishDelayMs;
+    uint64_t statsIntervalInSeconds;
```

Comment:
why not unsigned long like the items above.  unsigned int would be even better, except for the inconsistency.